Marlin/src/module/temperature.cpp

```diff
@@ -1176,6 +1176,14 @@ void Temperature::init() {
       setPwmFrequency(FAN2_PIN, 1); // No prescaling. Pwm frequency = F_CPU/256/8
     #endif
   #endif
+  
+  #if ENABLED(USE_CONTROLLER_FAN)
```

Comment:
This should be `#if HAS_CONTROLLER_FAN` to mirror the others.

Reply:
Then i would have to specifically define HAS_CONTROLLER_FAN  in a separate file. As far i can tell the ENABLED is a other version of it without the hassle to redefine it via a check.